internal/olm/operator/registry/index_image.go

```diff
@@ -59,20 +66,40 @@ const (
 	registryPodNameAnnotation = operatorFrameworkGroup + "/registry-pod-name"
 )
 
+const (
+	SchemaChannel  = "olm.channel"
+	SchemaPackage  = "olm.package"
```

Comment:
Let's unexport `SchemaPackage` as it's only being used here.

```suggestion
	schemaPackage  = "olm.package"
```